lib/node/closure-compiler.js

```diff
@@ -34,7 +34,7 @@ var contribPath = path.dirname(compilerPath) + '/contrib';
  * @param {Array<String>=} extraCommandArgs
  */
 function Compiler(args, extraCommandArgs) {
-  this.commandArguments = extraCommandArgs || [];
+  this.commandArguments = extraCommandArgs.slice() || [];
```

Comment:
This won't work if `extraCommandArgs` is `undefined`. I think this line can just be:
```js
this.commandArguments = (extraCommandArgs || []).slice();
```